mirdata/datasets/egfxset.py

```diff
@@ -0,0 +1,383 @@
+    ),
+    "digitalDelay": download_utils.RemoteFileMetadata(
+        filename="Digital-Delay.zip",
+        url="https://zenodo.org/record/7044411/files/Digital-Delay.zip?download=1",
+        checksum="4a25d57bcb0083667bade7b3c42460bc",
+    ),
+    "flanger": download_utils.RemoteFileMetadata(
+        filename="Flanger.zip",
+        url="https://zenodo.org/record/7044411/files/Flanger.zip?download=1",
+        checksum="f3f7b39c895a400d35c5b1314a1122bd",
+    ),
+    "hallReverb": download_utils.RemoteFileMetadata(
+        filename="Hall-Reverb.zip",
+        url="https://zenodo.org/record/7044411/files/Hall-Reverb.zip?download=1",
+        checksum="c173bebdcbed50d4bc8803e0b30d6517",
+    ),
+    "phaser": download_utils.RemoteFileMetadata(
+        filename="Phaser.zip",
+        url="https://zenodo.org/record/7044411/files/Phaser.zip?download=1",
+        checksum="1842e2643dd34d7285a77506ca540df3",
+    ),
+    "plateReverb": download_utils.RemoteFileMetadata(
+        filename="Plate-Reverb.zip",
+        url="https://zenodo.org/record/7044411/files/Plate-Reverb.zip?download=1",
+        checksum="abbcc68d692f323e8af6aeb8d478c40d",
+    ),
+    "rat": download_utils.RemoteFileMetadata(
+        filename="RAT.zip",
+        url="https://zenodo.org/record/7044411/files/RAT.zip?download=1",
+        checksum="afe9fc757a51d04126c23159706f4e8e",
+    ),
+    "spring-Reverb": download_utils.RemoteFileMetadata(
+        filename="Spring-Reverb.zip",
+        url="https://zenodo.org/record/7044411/files/Spring-Reverb.zip?download=1",
+        checksum="21afc47594ed8f37db008f313e50c634",
+    ),
+    "sweepEcho": download_utils.RemoteFileMetadata(
+        filename="Sweep-Echo.zip",
+        url="https://zenodo.org/record/7044411/files/Sweep-Echo.zip?download=1",
+        checksum="ea6dda440e9af6a19173facdf2bf17ac",
+    ),
+    "tapeEcho": download_utils.RemoteFileMetadata(
+        filename="TapeEcho.zip",
+        url="https://zenodo.org/record/7044411/files/TapeEcho.zip?download=1",
+        checksum="77adf4a6a8ed4eb566b2b8e77735c7dc",
+    ),
+    "tubeScreamer": download_utils.RemoteFileMetadata(
+        filename="TubeScreamer.zip",
+        url="https://zenodo.org/record/7044411/files/TubeScreamer.zip?download=1",
+        checksum="b9c46ed65037d0bd17bdf82dc3125beb",
+    ),
+    "metadata": download_utils.RemoteFileMetadata(
+        filename="egfxset_metadata.csv",
+        url="https://zenodo.org/record/7044411/files/egfxset_metadata.csv?download=1",
+        checksum="ec8d160fe79469c7de8cad528d7d35e1",
+    ),
+}
+
+LICENSE_INFO = "Creative Commons Attribution 4.0 International"
+
+
+class Track(core.Track):
+    """EGFxSet Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        audio_path (str): path to the track's audio file
+        stringfret_tuple (list): an array with the tuple of the note recorded
+        note (str): the notename of the file (i.e. D1,Eb4, etc.)
```

Comment:
`mirdata` has a `NoteData` class, you should use that instead and indicate the units as `note_name`